compiler/ras/DebugCounter.cpp

```diff
@@ -361,18 +361,19 @@ TR::DebugCounter *TR::DebugCounterGroup::findCounter(const char *nameChars, int3
    {
    if (nameChars == NULL)
       return NULL;
+
+   // Null terminate the name
```

Comment:
I don't think the "Null terminate" comment is adding much value.  This is a common pattern and its fairly obvious from the code what it's up to.

This did, however, draw my eye to this code and the `alloca` call seems unsafe from a security (stack overflow) perspective if the debug counter name and length passed in can be arbitrary and specified by a user.  I'll create an issue to investigate this.  This appears to be the only `alloca` usage in the OMR compiler code.  I think there are safer alternatives to use.